src/implementations/twig/components/rating-field/README.md

```diff
@@ -8,34 +8,25 @@ npm install --save @ecl/twig-component-rating-field
 
 ### Parameters:
 
-- **"items"** (array) (default: [])
-  - "id" (string) (default: '')
-  - "name" (string) (default: '')
-  - "value" (string) (default: '')
-  - "disabled" (boolean) (default: false)
-  - "checked" (boolean) (default: false)
-  - "required" (boolean) (default: false)
-  - "helper_id" (string) (default: '')
-  - "helper_text" (block) (default: '')
-  - "label" (block) (default: '')
+- **"id"** (string) (default: 'rating'): unique id for the rating field
+- **"items"** (array) (default: []): array of radio_button
 - **"name"** (string) (default: '')
```

Comment:
we are setting a default  "rating" for this parameter, i wonder whether we really need to get two separated parameter for setting ids and names of the inputs, couldn't we just use one and use it as such for the name of the input group and for the ids with an appended string..?

Reply:
I'd rather keep both name and id separated, as they are not exactly the same thing. Also that's how we defined the checkboxes/radio.